src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+
+    String getPrintableString();
```

Comment:
You should have a header comment here, so that other developers know how to properly implement this method.